From ae6de051eb10f416bd235d451da1d9d38d2008b0 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Wed, 7 Feb 2024 10:45:42 +0100
Subject: [PATCH] arm64: dts: st: enable secure arm-wdt watchdog on
 stm32mp235f-dk

Enable the watchdog and define the default timeout to 32 seconds.

Change-Id: Ic35cbbbfcca9bb063162ed90030dce5a9be24ede
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/358025
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp235f-dk.dts | 5 +++++
 1 file changed, 5 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp235f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp235f-dk.dts
@@ -50,6 +50,11 @@
 	};
 };
 
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &sdmmc1 {
 	pinctrl-names = "default", "opendrain", "sleep";
 	pinctrl-0 = <&sdmmc1_b4_pins_a>;
